packages/mongodb-chatbot-server/src/processors/makeVerifiedAnswerGenerateResponse.test.ts

```diff
@@ -24,6 +27,29 @@ describe("makeVerifiedAnswerGenerateResponse", () => {
     },
   ] satisfies GenerateResponseReturnValue["messages"];
 
+  const streamVerifiedAnswer: StreamFunction<{
+    verifiedAnswer: VerifiedAnswer;
+  }> = async ({ dataStreamer, verifiedAnswer }) => {
+    dataStreamer.streamData({
+      type: "metadata",
+      data: {
+        verifiedAnswer: {
+          _id: verifiedAnswer._id,
+          created: verifiedAnswer.created,
+          updated: verifiedAnswer.updated,
+        },
+      },
+    });
+    dataStreamer.streamData({
+      type: "delta",
+      data: verifiedAnswer.answer,
+    });
+    dataStreamer.streamData({
+      type: "references",
+      data: verifiedAnswer.references,
+    });
+  };
```

Comment:
@mongodben  in reference to that case I mentioned about "not receiving a finished event when references are returned", do you think that's related to this? Should this stream a finished event?